### DISCUSSION TITLE
Options crash

### DISCUSSION THREAD

**Discussion by FranzSt** [rsjaffe/MIDI2LR#595] [Problems]
Hi, I know this topic has been brought up before but I've been plaguing myself for two days and would like to set my presets with an X
![Uploading image.jpg…]()
 Touch but under the Settings add-on module when I click on Options or Development Presets Settings I get an error message. I am working with Mac OS 10.12, lightroom CC2015 and Midi2LR 3.5.0.3. please can you help me?

**Comment by rsjaffe** [comment]
What error message are you receiving, and please post your log file: ~/Library/Logs/MIDI2LR/MIDI2LR.log.

**Comment by FranzSt** [comment]
Hi, thank you for your reply. This is a picture of the error message
![BB8BD502-1021-411F-B972-9FBCB51C69D5](https://user-images.githubusercontent.com/96378095/146690369-39163dc5-8441-4500-a698-69fdcea1f956.jpeg)
The log file I will send tomorrow

**Reply by FranzSt** [reply]
[Midi2LR 3.4.10.txt](https://github.com/rsjaffe/MIDI2LR/files/7741615/Midi2LR.3.4.10.txt)
This ist thr error report

**Reply by rsjaffe** [reply]
You're also running a broken installation: your MIDI2LR app has a different version from your plugin. I'd make sure you have only one MIDI2LR.lrplugin directory on your computer, then reinstall 3.5.0.3.

**Reply by FranzSt** [reply]
Thank you for your message. I only have one plugin in the folder - I took the screenshot while still testing the old version. But the error is the same for all versions.

**Comment by rsjaffe** [comment]
please post your log file: ~/Library/Logs/MIDI2LR/MIDI2LR.log.

**Reply by FranzSt** [reply]
Oh sorry, here it is - thank you for your help
[MIDI2LR.log](https://github.com/rsjaffe/MIDI2LR/files/7748245/MIDI2LR.log)
p

**Reply by FranzSt** [reply]
@rsjaffe do you have an idea? Please can you help me?

**Comment by rsjaffe** [comment]
Try this:
1. Install version 3.5.0.3
2. Unzip this zip file: [Info.zip](https://github.com/rsjaffe/MIDI2LR/files/7776308/Info.zip)
3. Put the contents (Info.lua and PreReset.lua) into your MIDI2LR.lrplugin directory.
4. Start up Lightroom
4. This adds a new menu item under File → Plug-in Extras: Reset preferences (it won't be translated into your language).
5. Select reset preferences.
6. Exit Lightroom
7. Restart Lightroom
8. Should be fixed.
